src/cmd-buildextend-metal

```diff
@@ -173,7 +173,7 @@ if [ -z "${use_anaconda}" ]; then
         ref_arg=${commit}
     fi
 
-    runvm -drive "if=virtio,id=target,format=${image_format},file=${path}.tmp" -- /usr/lib/coreos-assembler/create_disk.sh /dev/vda "$ostree_repo" "${ref_arg}" "${ostree_remote}" /usr/lib/coreos-assembler/grub.cfg "$name" "${save_var_subdirs}" "\"$kargs\""
+    runvm -drive "if=virtio,id=target,format=${image_format},file=${path}.tmp" -- /usr/lib/coreos-assembler/create_disk.sh /dev/vda "${img}" "$ostree_repo" "${ref_arg}" "${ostree_remote}" /usr/lib/coreos-assembler/grub.cfg "$name" "${save_var_subdirs}" "\"$kargs\""
```

Comment:
Shouldn't we be passing `${build}` here?

Reply:
I think this is a "why not both?" situation - the imageid carries the platform and OS name too; it's completely unambiguous.  But I can see the pure buildid being useful too so people don't have to parse the imageid.  Added both.